collector/prom-native-ceph/README.md

```diff
@@ -0,0 +1,98 @@
+# Ingest Ceph metrics using the OpenTelemetry Collector
+
+## Overview
+
+ Ceph natively exposes a Prometheus endpoint and the OpenTelemetry Collector has a [Prometheus receiver][otel-prom-receiver] that can be used to scrape its Prometheus endpoint. This directory contains an example showing how to configure Ceph and the Collector to send metrics to Lightstep Observability.
+
+## Prerequisites
+
+* Docker
+* Docker Compose
+* A Lightstep Observability [access token][ls-docs-access-token]
+
+## How to run the example
+
+* Export your Lightstep access token
+  ```
+  export LS_ACCESS_TOKEN=<YOUR_TOKEN>
+  ```
+* Run the docker compose example
+  ```
+  docker compose up -d
+  ```
+* Enabling Prometheus
+  ```
+  docker exec mon1 ceph --cluster ceph mgr module enable prometheus
+  ```
+* Disabling Prometheus
+  ```
+  docker exec mon1 ceph --cluster ceph mgr module disable prometheus
+  ```
+* Check Ceph Health
+  ```
+  docker exec mon1 ceph --cluster ceph -s
+  ```
+* Stop the cluster
+  ```
+  docker-compose down
+  ```
+
+### Explore Metrics in Lightstep
+
+See the [Ceph Telemetry Docs][ceph-docs-prometheus] for comprehensive documentation on metrics emitted and the [dashboard documentation][ls-docs-dashboards] for more details.
+
+### Explore the Ceph Example
+
+* Access Vault UI [http://localhost:8280](http://localhost:8280).
+
+
+## Configure Ceph
+
+- Prometheus Module provides a Prometheus exporter to pass on Ceph performance counters from the collection point in ceph-mgr. Ceph-mgr receives MMgrReport messages from all MgrClient processes (mons and OSDs, for instance) with performance counter schema data and actual counter data, and keeps a circular buffer of the last N samples. This module creates an HTTP endpoint (like all Prometheus exporters) and retrieves the latest sample of every counter when scraped.
+
+```sh
+$ sudo ceph mgr module enable prometheus
+```
+```sh
+$ sudo ceph mgr module disable prometheus
+```
+
+Enabling Prometheus Output  
```

Comment:
```suggestion
## Enabling Prometheus Output  
```
This looks like it's meant to be a heading to me. I believe that this is the appropriate level, but check it out with what you intended.